src/System.Net.Security/tests/FunctionalTests/DummyTcpServer.cs

```diff
@@ -140,40 +143,25 @@ private void OnAccept(IAsyncResult result)
                 if (useSsl)
                 {
                     state = new ClientState(client, sslEncryptionPolicy);
-                    Logger.LogInformation("Server: starting SSL authentication.");
+                    _log.WriteLine("Server: starting SSL authentication.");
 
 
                     SslStream sslStream = null;
                     X509Certificate2 certificate;
+                    certificate = new X509Certificate2("DummyTcpServer.pfx");
```

Comment:
If we have to use binary files, they need to be added to the corefx-testdata depot and not the corefx repo.

Otherwise, I agree we should Base64 encode, etc. the needed data.